Source/IntegrationTests/TestFiles/LitTests/LitTest/dafny0/SubsetTypes.dfy

```diff
@@ -1,4 +1,4 @@
-// RUN: %exits-with 4 %build "%s" --relax-definite-assignment --allow-axioms > "%t"
+// RUN: %exits-with 4 %verify "%s" --performance-stats=10 --relax-definite-assignment --allow-axioms > "%t"
```

Comment:
It would be worth documenting this attribute some where, wouldn't it?

Reply:
I meant to make it hidden, as a language developer option, but forgot to do so. I've done that now.